### PR TITLE
feat: centralize background image handling and clean up assets

### PR DESCRIPTION
**Pull request by BigBen-7**

### CODE REVIEW

**Review by whateverfw — CHANGES_REQUESTED** [review]
Individual pages should not have bg files, it should only be set in a wrapper (.App) once - for desktop and mobile.

Remove background from all other .css files, along with image assets that should not be used anymore

**Comment by whateverfw** [comment]
@BigBen-7 Also please remove merge conflicts (I believe you can accept all changes from main and then remove background in those css)

**Comment by djeck1432** [comment]
@BigBen-7 Please, resolve conflict

File: frontend/src/globals.css
Comment (whateverfw) [inline frontend/src/globals.css:1]: mobile background also should be here and removed from individual pages.

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Comment by whateverfw** [comment]
> Individual pages should not have bg files, it should only be set in a wrapper (.App) once - for desktop and mobile.
> 
> Remove background from all other .css files, along with image assets that should not be used anymore

@BigBen-7 it is still not resolved, I don't see old background files to be removed, and background still exists on page levels

File: frontend/src/globals.css [hunk 1]
```diff
@@ -98,3 +98,19 @@ li {
...
+
+.App {
+  background: url('../public/desktop-background.png') no-repeat;
+  background-size: cover;
+  min-height: 100vh;
+  box-sizing: border-box;
+}
+
+@media (max-width: 550px) {
+  .withdrawall-wrapper {
```
Comment (whateverfw) [inline frontend/src/globals.css:110]: why this applied to withdrawall-wrapper? 
please fix and test next time before pushing, thanks

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Review by whateverfw — APPROVED** [review]
lgtm

**Review by djeck1432 — APPROVED** [review]
looks good to me

**Pull request merged**